pages/04.Reference/13.secrets/docs.en.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Secrets
+published: true
+date: '18-06-2020 10:40'
+taxonomy:
+    category:
+        - docs
+---
+
+## Overview
+
+SysEleven Stack uses the OpenStack component Barbican for secret storage. We introduced the Barbican secret storage first of all to provide a safe way to store SSL certificates and private keys for [Octavia Load balancer as a Service](../08.network/02.lbaas/docs.en.md).
+
+!! Barbican is part of the Octavia public beta phase. This means we invite you to test the Barbican secret storage, but we do not recommend you to use it for production workloads yet.
+
+The barbican secret storage is part of our global region. This means, similar to [Keystone (Identity and Access)](../01.identity-and-access/docs.en.md) and [Designate (DNSaaS)](../07.dns/docs.en.md), there is one API for all regions.
+
+Barbican feature                     | Supported
+-------------------------------------|-------------
+Secret storage and metadata          | Yes
+Secret containers                    | Yes
+Secret consumers                     | Yes
+Access control lists                 | Yes
+Certificate orders                   | No
+
+## Secret storage and metadata
+
+All secrets are transferred and stored fully encrypted at all times. Metadata may not be stored fully encrypted.
+
+## Secret containers
+
```

Comment:
Maybe "containers of secrets" or something like that here? As it is, it sounds like the containers are secret.